examples/configGeneration.md

```diff
@@ -44,10 +46,12 @@ commonLabels:
 resources:
 - deployment.yaml
 - service.yaml
-configMapGenerator:	
-- name: the-map	
-  literals:	
-    - altGreeting=Good Morning!	
+configMapGenerator:
+- name: the-map
+  KVSources:
+  - name: literals
+    args:
```

Comment:
I may have misunderstood,  `args` can be a list of files OR individual properties? Can they be combined?

KVSources:
  name: stuff
  args:
    altGreeting=Bonjour!
    /configs/configfiles

KVSources ->name represents a `type` of configuration (file, properties, env vars) or arbitrary name?

Reply:
`name` is the plugin name. In this case it is one of the builtin plugins (files|envfiles|literals).

Reply:
normally you would need `pluginType` but it defaults to builtins.